### PR TITLE
Set overflow-behaviour for Designer Page

### PR DESCRIPTION
**Pull request by guptaharsh25**
[AB#34528](https://microsoftgarage.visualstudio.com/002806e2-ebaa-4672-9d2e-5fe5d29154ef/_workitems/edit/34528)

This will help with ensuring there's no scrolling on the designer page.

### CODE REVIEW

File: private-templates-service/client/src/components/Designer/styled.tsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ export const OuterDesignerWrapper = styled.main`
   display: flex;
   flex-direction: column;
   height: 100%;
+  overflow-behaviour: contain;
```
Comment (beervoley) [inline private-templates-service/client/src/components/Designer/styled.tsx:7]: I think it's a typo: `overscroll-behavior` - it should be

Comment (guptaharsh25) [reply private-templates-service/client/src/components/Designer/styled.tsx]: my bad, fixed

File: private-templates-service/client/src/components/Designer/styled.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ export const OuterDesignerWrapper = styled.main`
   display: flex;
   flex-direction: column;
   height: 100%;
-  overflow-behaviour: contain;
+  overscroll-behaviour: contain;
```
Comment (gracez72) [inline private-templates-service/client/src/components/Designer/styled.tsx:7]: is it supposed to be behavior?

Comment (guptaharsh25) [reply private-templates-service/client/src/components/Designer/styled.tsx:7]: i think so, `overscroll-behavior: contain;` sets the inner elements to be unscrollable.
https://developer.mozilla.org/en-US/docs/Web/CSS/overscroll-behavior

Comment (gracez72) [reply private-templates-service/client/src/components/Designer/styled.tsx:7]: I mean without the u in behaviour!

Comment (guptaharsh25) [reply private-templates-service/client/src/components/Designer/styled.tsx]: haha whoops, fixed

**Pull request merged**